app/controllers/registers_controller.rb

```diff
@@ -48,19 +48,15 @@ def get_field_definitions
   def show
     @register = Spina::Register.find_by_slug!(params[:id])
 
-    unless @register.populated
-      populate_register(@register)
-    end
+    populate_register(@register)
```

Comment:
since you are no longer using `register.populated` you could add a migration to remove this column.

Reply:
Good point.

Reply:
Done.

Reply:
should this column be removed from `schema.rb` also?

Reply:
Yep, it has been.